runtime/lua/vim/treesitter.lua

```diff
@@ -248,62 +256,55 @@ function M.get_captures_at_pos(bufnr, row, col)
   return matches
 end
 
---- Returns a list of highlight capture names under the cursor
+--- Shows the list of highlight capture names under the cursor
 ---
 ---@param winnr (number|nil) Window handle or 0 for current window (default)
----
----@return string[] List of capture names
-function M.get_captures_at_cursor(winnr)
+function M.show_captures_at_cursor(winnr)
```

Comment:
Would it make sense to make this `show_captures_at_pos` that defaults to the cursor position when arguments are omitted, just like the other ones? Seems like that could be more generally useful.

Most likely getting captures at the cursor position will be the most common use case, but if it's easy to support showing captures at an arbitrary position then why not?

Reply:
What would be the use case? (Adding arbitrary position would complicate/duplicate the code somewhat, so I'm a bit hesitant.)

Reply:
Would it? I think you would just add a `row` and `col` parameter like you have for `get_captures_at_pos` and forward those directly to `get_captures_at_pos` on line 267.

Reply:
Ah, you're right, I was overcomplicating this.

But still, it would be helpful for me to understand when you would like to "pretty print" to the message area the capture at an arbitrary position?

Reply:
I don't have a use case on top of mind. But if it's easy to make the interface more flexible while also keeping the common case simple that seems like a win to me.

The symmetry with `get_captures_at_pos` is nice too.

Reply:
Conversely, do you have a use case for _getting_ the captures/node at the current cursor position?